docs/tutorials/shallow-join.rst

```diff
@@ -0,0 +1,359 @@
+Visualizing Analytics in Jupyter
```

Comment:
Yeah, if there's an existing name, let's use it.  Explaining that we are forming an inner join with a covering geo set is useful.